src/config.ts

```diff
@@ -68,9 +69,10 @@ export function firebaseConfig(): firebase.AppOptions | null {
 
   // Could have Runtime Config with Firebase in it as an ENV location or default.
   try {
-    const path =
-      process.env.CLOUD_RUNTIME_CONFIG || '../../../.runtimeconfig.json';
-    const config = require(path);
+    const configPath =
+      process.env.CLOUD_RUNTIME_CONFIG ||
+      path.join(process.env.PWD, '.runtimeconfig.json');
```

Comment:
As written this is technically a breaking change. To make it not a breaking change we'd have to try the relative require first (ideally by checking to see if the file exists instead of relying on a caught exception) and then fall through to `$PWD` based if that doesn't work.

Reply:
I'm not sure that it is a breaking change. For trigger parsing, `PWD` is always the temp folder which is a copy of the functions folder. For in the GCF runtime, `PWD` is the root of the uploaded source code. We have always required that `package.json` be at the root of the functions directory, otherwise the function will not deploy. Therefore the location of `node_modules/firebase-functions` is always the same. So this change doesn't change any behavior.